tests/video-handler.test.ts

```diff
@@ -1,7 +1,7 @@
 import assert from 'node:assert';
 import { describe, it } from 'node:test';
 
-import videoHandler, { callHandler } from '../src/video-handler.js';
```

Comment:
this threw an error somewhere. 
it's best to avoid mixed named and default imports